components/item-act.js

```diff
@@ -271,7 +271,7 @@ export function useZap () {
     const item = cache.readFragment({
       id: `Item:${id}`,
       fragment: gql`
-        fragment ItemMeSats on Item {
+        fragment ItemMeSatsZap on Item {
           meSats
```

Comment:
these were giving me duplicate fragment name warnings ... the fragment names are really used so I gave them slightly different names